packages/dds/legacy-dds/src/array/interfaces.ts

```diff
@@ -40,6 +29,13 @@ export type FullyReadonly<T> = {
  * @alpha
  */
 export interface ISharedArrayEvents extends ISharedObjectEvents {
+	/**
+	 *
```

Comment:
Can we add a description of what this _does_? Same for the other newly documented listeners.

Thank you for adding documentation!!